javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,19 @@
+var fullName = 'Julie Dumbrique';
+var isCool = true;
+var totalPets = 1;
+
+console.log(fullName);
+console.log(isCool);
+console.log(totalPets);
+
+console.log(typeof 'Julie Dumbrique');
+console.log(typeof true);
+console.log(typeof 1);
```

Comment:
You shouldn't be logging out the value you assigned to the variable directly, you should be logging out the variable itself.